frontend/dockerfile/parser/directives.go

```diff
@@ -13,12 +13,14 @@ import (
 
 const (
 	keySyntax = "syntax"
+	keyCheck  = "check"
```

Comment:
@colinhemmings I'm not sure if "check" works in here.

```
#syntax=docker/dockerfile
#check=skip=StageNameCasing

FROM alpine
```

Reply:
I've made a comment on the proposal related that `rule` would be a better term to use here